### ISSUE TITLE
Add VCF files necessary to generate plots to the repo

### ISSUE THREAD

**Issue by danvk** [hammerlab/concordance#16]

**Comment by ihodes** [comment]
Are you thinking some in the /test directory?

**Comment by danvk** [comment]
No, I'm thinking whatever you ran to generate concordance-data.js.

**Comment by ihodes** [comment]
I was actually planning on removing that sample data (or moving it to /test), as it's outside the scope of the project. This is meant to run on arbitrary VCFs, so adding some VCFs seems a bit outside the scope, no?

**Comment by danvk** [comment]
That's fine, so long as they're in a known place. I just want to be able to
regenerate the data myself.

But so long as the files aren't large or confidential, I don't see much
downside to including them. It's nice to have a few example inputs to make
it easy to get started.
